lib/src/models/document.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:firedart/src/generated/google/firestore/v1/document.pb.dart'
+    as fs;
+import 'package:firedart/src/models/document_reference.dart';
+import 'package:firedart/src/repository/firestore_gateway.dart';
+import 'package:firedart/src/util/firestore_encoding.dart';
+
+/// A [Document] contains data read from a document in your Firestore
+/// database.
+/// The data can be extracted with the data property.
```

Comment:
```suggestion
/// The data can be extracted with the data property.
```
Can you please clarify what you mean by `data` property here?